tools/hanabi_game_info.py

```diff
@@ -0,0 +1,12 @@
+class GameInfo:
+    def __init__(self):
```

Comment:
yup, I was thinking about builder pattern when I made this change. Since you can directly set fields, I was getting the effect with no additional code. I can do named arguments and default values, best of all worlds even if I'm not inclined to use any arguments